### ISSUE TITLE
Cleaning up a folder

### ISSUE THREAD

**Issue by alidan** [axcore/tartube#332]
Ok, I want to keep the files, I have my own sorting method for them, and I use tartube mostly as a bui for downloading, which its great at.

I have 2 folders which I use in it, one video which downloads audio and video, and one music which just gets audio. in older versions of the program, I was able to clear out both of these folders, but now it kind of seems like i'm stuck with files making it a bit difficult to see files that were just added, music having around 40 files and video having around 500

is there any way in program to empty out the folders in program without touching the files?

**Comment by axcore** [comment]
Sorry, I don't understand what you're trying to achieve. How did you "clear out both of these folders" in older versions? Which files were you able to remove? Which files are you not able to remove now?

**Comment by alidan** [comment]
ok, lets use the 'music' folder for this example
It has 22 entries. 
![python3_2021-09-14_17-33-26](https://user-images.githubusercontent.com/2681758/133342640-a77c21cc-2fe2-46e1-bfdf-76a8f9030740.png)
In older versions I was able to go here
![python3_2021-09-14_17-34-10](https://user-images.githubusercontent.com/2681758/133342756-3cebca9a-4fac-43df-b3e7-50cc04029695.png)
and hit this 
![python3_2021-09-14_17-34-21](https://user-images.githubusercontent.com/2681758/133342768-e9cbd85e-fe38-41f6-88e2-f957503d7509.png)

and it would clean the files out of the folder, there were a few times files were stubborn with removing themselves, and moving the downloaded file out of folder and deleting them in tartube would get rid of them, but that's not working anymore. 

Ok I just deleted the files one at a time and it was working, restarted it and some came back but some were gone and kept at this for a while, the files are now removed from the list, not sure why it worked now because I did this prior to submitting the issue. 

with that said, "just empty the folder in this list" didn't work and I know that use to, I have some things I want to download so ill get that done something this week and see if the way it works changed now that the prior files are now gone.

**Comment by alidan** [comment]
ok, downloaded a small short list of videos, none of them will go away with 'just empty the folder in this list' however they are all now consistently going away with deleting them, not necessarily the idea solution, but it works.

**Comment by axcore** [comment]
Sorry for the delay in responding. As I've had to write on every other issue, I had no internet since last summer.

I think the reason for your confusion is that the text in the window is gibberish. I have re-written it like this:

![emptyfolder](https://user-images.githubusercontent.com/37951926/162141908-f1d447b9-4c0c-4a34-8871-fbf2fbcd70ce.png)

Now it should be clear, that there are two choices:

1. Actually delete files on your computer
2. Don't delete any files on your computer

I have checked that both options are working correctly. However, there is a small complication. When you select **Delete files on your computer**, Tartube tells your computer to delete the whole directory/folder. There is no guarantee that your computer will allow that to happen (for example, if some of the files in the directory/folder are protected, or if the hard drive itself is physically damaged.)

The new code will be in the next release.

By the way, there is a second way to tidy up your filesystem: right-click the folder, and select **Folder actions > Tidy up folder**, and you will see this window:

![tidyup](https://user-images.githubusercontent.com/37951926/162143025-ae28ef33-4917-497c-aa6f-ff02cae62bd7.png)
